### PR TITLE
Add check for the minimal docker/-compose version

### LINKED ISSUE #253: Identify and show errors for old unsupported version of docker/docker-compose
On linux `docker-compose` gets installed separately from `docker` and can be very old.
In such case `sourced-ce` doesn't work and produces either strange or no errors at all.

We need to identify such old versions and ask a user to update the dependencies.

### PR DESCRIPTION
**Pull request by smacker**
Fix: #253

Signed-off-by: Maxim Sukharev <max@smacker.ru>

---

* [x] I have updated the CHANGELOG file according to the conventions in [keepachangelog.com](https://keepachangelog.com)
* [ ] This PR contains changes that do not require a mention in the CHANGELOG file

### CODE REVIEW

**Review by dpordomingo — COMMENTED** [review]
Could you mention these min versions in our docs?
https://github.com/src-d/sourced-ce/blob/master/docs/quickstart/1-install-requirements.md

File: cmd/sourced/compose/compose.go [hunk 1]
```diff
@@ -149,3 +186,59 @@ func RunWithIO(ctx context.Context, stdin io.Reader, stdout, stderr io.Writer, a
...
+	}
+	v.Patch, err = strconv.ParseUint(string(submatches[3]), 10, 64)
+	if err != nil {
+		return nil, fmt.Errorf("can't parse docker version")
+	}
+
+	return v, nil
+}
+
+func getDockerComposeVersions(bin string) (*semver.Version, error) {
```
Comment (se7entyse7en) [inline cmd/sourced/compose/compose.go]: Why plural?

**Review by se7entyse7en — CHANGES_REQUESTED** [review]
Maybe a typo.

**Comment by se7entyse7en** [comment]
Regarding the documentation, I don't see it as strictly required, because the error would be very clear. But if you prefer it's ok.

File: cmd/sourced/compose/compose.go (hunk 1 above)
Comment (smacker) [reply cmd/sourced/compose/compose.go]: typo, thanks!

**Comment by smacker** [comment]
Fixed.

**Comment by dpordomingo** [comment]
Given that there're people installing the tool to work offline, I think it could be better if we list the known dependencies (as they're not obvious)

**Comment by se7entyse7en** [comment]
> Given that there're people installing the tool to work offline

good point! I didn't think about it 👍

**Review by se7entyse7en — APPROVED** [review]

**Comment by smacker** [comment]
Versions were mentioned in the docs, the code changes are approved. Merging.

**Pull request merged**